tests/unit/lib/intent-parser/index_test.js

```diff
@@ -61,7 +61,11 @@ describe('intent-parser', function() {
         return intentParser.parse(sentence).then((result) => {
           assert.deepEqual(result.recipients, parsed.recipients);
           assert.equal(result.action, parsed.action);
-          assert.equal(result.confirmation, parsed.confirmation);
+          if (typeof parsed.confirmation === 'string') {
```

Comment:
In the case of query, the confirmation can't just be a simple string because it needs a response from the server.
Is the user busy? If so, what's his activity? Depending on that the confirmation will differ.
For the creation of a query, the confirmation is always a static string, for a query the confirmation is a function.